app/controllers/stories_controller.rb

```diff
@@ -56,36 +56,39 @@ def update
   end
 
   def import
-
     if !params[:file].original_filename.ends_with?(".csv")
       flash[:error] = "Invalid File: Must be CSV"
-      redirect_to(@project) and return
+      redirect_to(@project) && return
```

Comment:
hey @gelsey, everything looks good, I'm just wondering if these changes from `and` to `&&` work as expected because the documentation states that we should use `and return` https://guides.rubyonrails.org/layouts_and_rendering.html#avoiding-double-render-errors.

I think the problem is prevented because we have `redirect_to(@project)` instead of `redirect_to @project`, I'm not sure if there's a quick way to test this manually just to verify.

Reply:
Hi @arielj,
I think you are right that `redirect_to(@project) && return` works as expected because we have `redirect_to(@project)`. I tested the import functionality manually with a non-CSV file, an empty CSV file, a CSV file that only had an id and title, and a correct CSV file, and for each input, the `redirect_to(@project) && return` works as expected. As an example, here is a screenshot that shows the redirection when I try to import a non-CSV file:
<img width="777" alt="Screen Shot 2021-05-27 at 11 32 07 AM" src="https://user-images.githubusercontent.com/6892410/119857403-85677100-bee1-11eb-8b73-cae67e96638d.png">